task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -6,12 +6,30 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Enter the first number"); 
+            Console.WriteLine("Enter the first number");
             int firstInputNumber = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter the second number");
             int secondInputNumber = int.Parse(Console.ReadLine());
+            int min = 0;
+            int max = 0;
 
-            for (int i = firstInputNumber; i <= secondInputNumber; i++)
+            if (firstInputNumber < secondInputNumber)
+            {
+                min = firstInputNumber;
```

Comment:
You have the same bodies in if condition and in the last else if condition. Maybe we coulde combine this conditions?

Reply:
I mean in the both cases you have body
```
{
    min = firstInputNumber;
    max = secondInputNumber;
}
```